modules/common/src/main/scala/surge/internal/akka/actor/ActorLifecycleManagerActor.scala

```diff
@@ -8,7 +8,7 @@ import surge.internal.akka.actor.ActorLifecycleManagerActor.Ack
 object ActorLifecycleManagerActor {
   case object Start
   case object Stop
-  case class Ack(success: Boolean)
+  case class Ack()
```

Comment:
I think we can just leverage the common `Ack()` here rather than a custom `ActorLifecycleManagerActor.Ack()` since they're identical now

Reply:
done